staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go

```diff
@@ -237,6 +246,19 @@ type MemoryStats struct {
 	MajorPageFaults *uint64 `json:"majorPageFaults,omitempty"`
 }
 
+// SwapStats contains data about memory usage
+type SwapStats struct {
+	// The time at which these stats were updated.
+	Time metav1.Time `json:"time"`
+	// Available swap memory for use.  This is defined as the memory limit - workingSetBytes.
```

Comment:
this is swap memory limit, right? Maybe need to add as a qualifier

Reply:
Not exactly. As documented, it's equivalent to `MemoryStats`'s `AvailableBytes` which is calculated by `<swap-limit> - <current-swap-usage>`.

On a second look the documentation can be improved a bit. I'll try to reshape it.